Luke-Mao/Luke-Mao.md

```diff
@@ -0,0 +1,14 @@
+---
+id: '9b0d2f13-16cf-eb11-bacc-002248152435'
+
+qualifications: ''
+
+quote: 'Stay hungry. Stay foolish.'
+quoteAuthor: 'Steve Jobs'
+---
+
+Luke is a senior software developer with more than three years of development experience. He has a broad technology stack, especially good at using React for web development.
```

Comment:
I would say "and is especially good at using React for web development." instead of " especially good at using React for web development."

Reply:
Done